### ISSUE TITLE
[Xcode10]monotouch-test failures on iOS 8.0 device

### ISSUE THREAD

**Issue by GouriKumari** [xamarin/xamarin-macios#4437]
### Steps to Reproduce
1.  Update System to Xamarin.iOS 11.99.1.44
2. Build and Execute monotouch-test on an iOS 8.0 device

### Expected Behavior
All tests must pass

### Actual Behavior
[FAIL] SystemSoundTest.TestCallbackPlayAlert : System.EntryPointNotFoundException : AudioServicesPlayAlertSoundWithCompletion
[FAIL] ImagePropertiesTest.CameraMaker :   MakerMinoltaDictionary
[FAIL] LocalSearchTest.EmptyRequest : Foundation.NSErrorException : Exception of type 'Foundation.NSErrorException' was thrown.
[FAIL] A_UIStackViewTest.InitWithFrameTest : ObjCRuntime.RuntimeException : Wrapper type 'UIKit.UIStackView' is missing its native ObjectiveC class 'UIStackView'

### Environment
```
Operating System: 
Mac OS X 10.13.5
Mono version: 5.12.0.277 (4e1474e9e33)  (2018-06-18 05:55:00 -04:00)
XA version: 9.0.0.8 (HEAD/481cf8a98) 
XI version: 11.99.1.44 (xcode10: c1ad3f78) 
XM version: 4.99.1.164 (c1ad3f78)  (2018-07-09 13:27:48 -04:00)
XVS version: 
XS version: 7.5.4.5 (f2dc16b)  (2018-07-11 07:22:25 -04:00)
[Device] Name: XQAiPhone6a, Identifier: 7e1725ec6020e1343572d00f31c092550f08e574, Version: 8.0.2, Arch: ARM64, FormFactor: iPhone, PairedPhone: 

```

### Build Logs

Build log: http://xqa.blob.core.windows.net/gist/report-f5810e7108524861b0d64e81bca4d97f.txt
Test+Device Log: http://xqa.blob.core.windows.net/gist/monotouchtest-ios03-12-067e1725ec6020e1343572d00f31c092550f08e57403-12-06-dea0bb484e3d43838c73ba33dd150b69.zip

### Example Project (If Possible)
xamarin-macios/tests/monotouch-test

**Comment by mandel-macaque** [comment]
I do not have a device with the correct iOS version. @dalexsoto @chamons do you guys have one around?

**Comment by chamons** [comment]
negative

**Comment by dalexsoto** [comment]
Nope, no iOS 8 here

**Comment by mandel-macaque** [comment]
I cannot reproduce the issue with the simulator. Trusting @GouriKumari with the results in a device.

**Comment by mandel-macaque** [comment]
@GouriKumari can we confirm the bug was closed.

**Comment by spouliot** [comment]
Closing, next round of test will verify this

**Comment by GouriKumari** [comment]
Verified with

**Comment by spouliot** [comment]
@mandel-macaque does not have a device with iOS8 so I'll take a look

**Comment by spouliot** [comment]
Fix merged with PR #4719

**Comment by GouriKumari** [comment]
Verified with XI Version: 11.99.3.601 (Visual Studio Community), Hash: e8e1d429
Branch: xcode10 and I do not see this issue on 8.1.3 device.

## Test Log: 
https://gist.github.com/GouriKumari/bdb9e6fa9815b69f71f5d858a1843e13
